src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-641.mdx

```diff
@@ -9,6 +9,8 @@ downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_
 ## Fixed in this release
 * Resolved an error when transform some libraries during build time.
   Detailed message: [ClassTransformer] transformArchive: java.util.zip.ZipException: duplicate entry: META-INF/MANIFEST.MF
+* Resolved a crash during runtime.
+  Detailed message: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/crypto/tink/daead/AesSivKeyManager
```

Comment:
Since this crash could apply to any class, remove ": _Failed resolution of: Lcom/google/crypto/tink/daead/AesSivKeyManager_"